core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -69,6 +70,8 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
   private var offsetCommitIntervalMs = 0
```

Comment:
not related to this PR. Can we update the comments above:
 "There are N mirror maker threads each having one KafkaConsumer instance" ?